platform/linux-generic/odp_ipsec.c

```diff
@@ -870,6 +938,9 @@ static ipsec_sa_t *ipsec_in_single(odp_packet_t pkt,
 		goto post_lifetime_err_cnt_update;
 	}
 
+	/* Truncate the ESN higher 32 bits */
```

Comment:
The comment is confusing since there is no truncation going on here.

Reply:
Ack, will fix it.